browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -54,6 +54,7 @@ class MarkdownNoteDetail extends React.Component {
   componentWillReceiveProps (nextProps) {
     if (nextProps.note.key !== this.props.note.key && !this.isMovingNote) {
       if (this.saveQueue != null) this.saveNow()
+      if (this.sendEventQueue != null) this.sendEditnoteEvent()
```

Comment:
Is there any probability that it becomes `null`? I assume it's `undefined` (unconfirmed).
(It returns true even if this.sendEventQueue is `undefined` because it's not a restricted equality.)